Regression tester regressed f84052

[start automated regression testsuite report]

Dear all,

this is the automated regression testsuite reporting a new regression between
- new revision: f84052e506e7e06a4356831d9a27f05754aca3ed  [regression testsuite results](https://cdash.dealii.org/index.php?project=deal.II&filtercount=1&showfilters=1&field1=revision&compare1=63&value1=f84052)
- old revision: ec996d327766145442f285d49e6362dc24a19233  [regression testsuite results](https://cdash.dealii.org/index.php?project=deal.II&filtercount=1&showfilters=1&field1=revision&compare1=63&value1=ec996d)

Summary:
- :angry: new test failures introduced for configuration [GNU-11.3.0-master-ubuntu-lts-22.04](https://cdash.dealii.org/build/2378) on site tester-tng
- :angry: new test failures introduced for configuration [GNU-12.2.0-master-debian-12](https://cdash.dealii.org/build/2380) on site tester-tng
- :angry: new test failures introduced for configuration [GNU-12.2.0-master-ubuntu-23.04](https://cdash.dealii.org/build/2381) on site tester-tng
- :angry: new test failures introduced for configuration [GNU-12.3.0-master-debian-13](https://cdash.dealii.org/build/2379) on site tester-tng
- :angry: new test failures introduced for configuration [Clang-16.0.6-master-autodetection-20230507](https://cdash.dealii.org/build/2386) on site tester-tos
- :angry: new test failures introduced for configuration [Clang-16.0.6-master-sanitizer-20230622](https://cdash.dealii.org/build/2395) on site tester-tos
- :angry: new test failures introduced for configuration [Clang-16.0.6-master-taskflow-20230708](https://cdash.dealii.org/build/2396) on site tester-tos
- :angry: new test failures introduced for configuration [GNU-13.1.1-master-64bit_indices-20230507](https://cdash.dealii.org/build/2394) on site tester-tos
- :angry: new test failures introduced for configuration [GNU-13.1.1-master-autodetection-20230507](https://cdash.dealii.org/build/2377) on site tester-tos
- :angry: new test failures introduced for configuration [GNU-13.1.1-master-minimal-20230116](https://cdash.dealii.org/build/2392) on site tester-tos

I have identified the following pull requests as possible candidates:
- <!-- Add the missing variants of TpetraWrappers::SparseMatrix::set(). -->#16732 (authored-by: @bangerth; merged-by: @kronbichler)
- <!-- Close the correct constraints objects. -->#16733 (authored-by: @bangerth; merged-by: @masterleinad)
- <!-- Also remove the specialization structure of MGTwoLevelTransferNonNested. -->#16730 (authored-by: @bangerth; merged-by: @kronbichler)
- <!-- Rename a few variables. -->#16735 (authored-by: @bangerth; merged-by: @kronbichler)
- <!-- MatrixFree: Split tensor product points and single point kernels -->#16736 (authored-by: @bergbauer; merged-by: @kronbichler)
- <!-- Colorize hyper rectangle with simplices -->#16713 (authored-by: @nmuch; merged-by: @kronbichler)


Notes:
 - The regression reported above is a subset of the following full set of regressions compared to the baseline:
   - :angry: test failures for configuration [GNU-11.3.0-master-ubuntu-lts-22.04](https://cdash.dealii.org/build/2378) on site tester-tng
   - :angry: test failures for configuration [GNU-12.2.0-master-debian-12](https://cdash.dealii.org/build/2380) on site tester-tng
   - :angry: test failures for configuration [GNU-12.2.0-master-ubuntu-23.04](https://cdash.dealii.org/build/2381) on site tester-tng
   - :angry: test failures for configuration [GNU-12.3.0-master-debian-13](https://cdash.dealii.org/build/2379) on site tester-tng
   - :angry: test failures for configuration [Clang-16.0.6-master-autodetection-20230507](https://cdash.dealii.org/build/2386) on site tester-tos
   - :angry: test failures for configuration [Clang-16.0.6-master-sanitizer-20230622](https://cdash.dealii.org/build/2395) on site tester-tos
   - :angry: test failures for configuration [Clang-16.0.6-master-taskflow-20230708](https://cdash.dealii.org/build/2396) on site tester-tos
   - :angry: test failures for configuration [GNU-13.1.1-master-64bit_indices-20230507](https://cdash.dealii.org/build/2394) on site tester-tos
   - :angry: test failures for configuration [GNU-13.1.1-master-autodetection-20230507](https://cdash.dealii.org/build/2377) on site tester-tos
   - :angry: test failures for configuration [GNU-13.1.1-master-minimal-20230116](https://cdash.dealii.org/build/2392) on site tester-tos
 - I will close and unpin this issue automatically once a full run is complete and compares cleanly to the baseline.
 - If closed I will reopen the issue if the testsuite run identified an additional regression compared to what I have reported so far.

[end automated regression testsuite report]

In fact, these are compile errors:
```
/scratch/users/testsuite/testsuite-FefJdjeG/dealii/tests/matrix_free/tensor_product_evaluate_01.cc: In function 'void test(unsigned int)':
/scratch/users/testsuite/testsuite-FefJdjeG/dealii/tests/matrix_free/tensor_product_evaluate_01.cc:73:34: error: 'evaluate_tensor_product_value_and_gradient' is not a member of 'dealii::internal'
   73 |       const auto val = internal::evaluate_tensor_product_value_and_gradient(
      |                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/scratch/users/testsuite/testsuite-FefJdjeG/dealii/tests/matrix_free/tensor_product_evaluate_01.cc:85:38: error: 'evaluate_tensor_product_value_and_gradient' is not a member of 'dealii::internal'
   85 |           const auto val = internal::evaluate_tensor_product_value_and_gradient(
      |                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/tests/matrix_free/tensor_product_evaluate_01.cc
+++ b/tests/matrix_free/tensor_product_evaluate_01.cc
@@ -28,7 +28,7 @@
 
 #include <deal.II/lac/vector.h>
 
-#include <deal.II/matrix_free/tensor_product_kernels.h>
+#include <deal.II/matrix_free/tensor_product_point_kernels.h>
 
 #include "../tests.h"
 

--- a/tests/matrix_free/tensor_product_evaluate_02.cc
+++ b/tests/matrix_free/tensor_product_evaluate_02.cc
@@ -28,7 +28,7 @@
 
 #include <deal.II/lac/vector.h>
 
-#include <deal.II/matrix_free/tensor_product_kernels.h>
+#include <deal.II/matrix_free/tensor_product_point_kernels.h>
 
 #include "../tests.h"
 

--- a/tests/matrix_free/tensor_product_evaluate_03.cc
+++ b/tests/matrix_free/tensor_product_evaluate_03.cc
@@ -29,7 +29,7 @@
 
 #include <deal.II/lac/vector.h>
 
-#include <deal.II/matrix_free/tensor_product_kernels.h>
+#include <deal.II/matrix_free/tensor_product_point_kernels.h>
 
 #include "../tests.h"
 

--- a/tests/matrix_free/tensor_product_evaluate_04.cc
+++ b/tests/matrix_free/tensor_product_evaluate_04.cc
@@ -29,7 +29,7 @@
 
 #include <deal.II/lac/vector.h>
 
-#include <deal.II/matrix_free/tensor_product_kernels.h>
+#include <deal.II/matrix_free/tensor_product_point_kernels.h>
 
 #include "../tests.h"
 

--- a/tests/matrix_free/tensor_product_evaluate_05.cc
+++ b/tests/matrix_free/tensor_product_evaluate_05.cc
@@ -28,7 +28,7 @@
 
 #include <deal.II/lac/vector.h>
 
-#include <deal.II/matrix_free/tensor_product_kernels.h>
+#include <deal.II/matrix_free/tensor_product_point_kernels.h>
 
 #include "../tests.h"
 

--- a/tests/matrix_free/tensor_product_evaluate_06.cc
+++ b/tests/matrix_free/tensor_product_evaluate_06.cc
@@ -27,7 +27,7 @@
 
 #include <deal.II/lac/vector.h>
 
-#include <deal.II/matrix_free/tensor_product_kernels.h>
+#include <deal.II/matrix_free/tensor_product_point_kernels.h>
 
 #include "../tests.h"
 

--- a/tests/matrix_free/tensor_product_evaluate_07.cc
+++ b/tests/matrix_free/tensor_product_evaluate_07.cc
@@ -27,7 +27,7 @@
 
 #include <deal.II/lac/vector.h>
 
-#include <deal.II/matrix_free/tensor_product_kernels.h>
+#include <deal.II/matrix_free/tensor_product_point_kernels.h>
 
 #include "../tests.h"
 

--- a/tests/matrix_free/tensor_product_evaluate_08.cc
+++ b/tests/matrix_free/tensor_product_evaluate_08.cc
@@ -26,7 +26,7 @@
 
 #include <deal.II/lac/vector.h>
 
-#include <deal.II/matrix_free/tensor_product_kernels.h>
+#include <deal.II/matrix_free/tensor_product_point_kernels.h>
 
 #include "../tests.h"
 

